content/getting-started/ruby.textile

```diff
@@ -0,0 +1,220 @@
+
+h2(#step-2). Step 2: Subscribe to a channel and publish a message
+
+Messages contain the data that a client is communicating, such as a short 'hello' from a colleague, or a financial update being broadcast to subscribers from a server. Ably uses channels to separate messages into different topics, so that clients only ever receive messages on the channels they are subscribed to.
+
+* Add the following lines to your @get_started@ method within the @EventMachine@ to create a channel instance and register a listener to subscribe to the channel. Then run it with @ruby get_started.rb@:
+
+```[ruby]
+# Get a channel instance
+channel = realtime_client.channels.get('my-first-channel')
+
+# Subscribe to messages on the channel
+channel.subscribe do |message|
+  puts "Received message: #{message.data}"
+end
+```
+
+* Use the Ably CLI to publish a message to your first channel. The message will be received by the client you've subscribed to the channel, and be logged to the console.
+
+```[sh]
+ably channels publish my-first-channel 'Hello!'
+```
+
+* In a new terminal tab, subscribe to the same channel using the CLI:
+
+```[sh]
+ably channels subscribe my-first-channel
+```
+
+Publish another message using the CLI and you will see that it's received instantly by the client you have running locally, as well as the subscribed terminal instance.
+
+To publish a message in your code, you can add the following line to your @get_started@ method after subscribing to the channel:
+
+```[ruby]
+channel.publish 'example', 'A message sent from my first client!'
+```
+
+h2(#step-3). Step 3: Join the presence set
+
+Presence enables clients to be aware of one another if they are present on the same channel. You can then show clients who else is online, provide a custom status update for each, and notify the channel when someone goes offline.
+
+* Add the following lines to your @get_started@ method to subscribe to, and join, the presence set of the channel. Then run it with @ruby get_started.rb@:
+
+```[ruby]
+# Subscribe to presence events on the channel
+channel.presence.subscribe do |member|
+  puts "Event type: #{member.action} from #{member.client_id} with the data #{member.data.inspect}"
+end
+
+# Enter the presence set
+channel.presence.enter("I'm here!")
+```
+
+* In the "dev console":https://ably.com/accounts/any/apps/any/console of your first app, attach to @my-first-channel@. Enter a @client_id@, such as @my-dev-console@, and then join the presence set of the channel. You'll see that @my-first-client@ is already present in the channel. In your terminal you'll see that an event was received when the dev console client joined the channel.
```

Comment:
Maybe I've missed something, but is there a reason that we use both the dev console and the CLI as the "other" Ably client within the same guide? Wouldn't one tool suffice?